com.doycho.euterpe.gtk.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id" : "com.doycho.euterpe.gtk",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.38",
```

Comment:
is there a reason this doesn't work with a new gnome sdk? 41 is the latest release and will be in security support much longer.

Reply:
also 3.38 is actually EOL.

Reply:
There is that. ;-)

Reply:
This is what Gnome Builder did when creating a new project and it seems to work just fine on all of my devices so I left it as it is :D Will give it a try with the latest SDK and I guess it will be alright too.

Reply:
Done. It seems to be working just fine with SDK 41.